project/Versions.scala

```diff
@@ -0,0 +1,13 @@
+object Versions {
+  val antlrVersion = "4.9.3"
+  val deltaVersion = "3.2.1"
+  val jacksonVersion = "2.15.0"
+  val log4jVersion = "2.24.3"
+  val openApiToolsJacksonBindNullableVersion = "0.2.6"
+  val orgApacheHttpVersion = "4.5.14"
+  val scala212 = "2.12.15"
+  val scala213 = "2.13.14"
+  val slf4jVersion = "2.0.13"
+  val spark3Version = "3.5.3"
+  val spark4Version = "4.0.0"
```

Comment:
nit: can you split the versions into separate groups:
- scala versions
- delta/spark
- other dependencies